### PR TITLE
Updated terms and conditions page on Apply

### PR DESCRIPTION
**Pull request by MaeveRoseveare**
## Context

For continuous applications, we need to update our T&Cs to reflect some new rules.

## Changes proposed in this pull request

Updated the T&Cs page.

To review the content, select the 'View deployment' button below on the PR. You'll see the candidate sign in screen. Select the 'Terms and conditions' link in the footer.

Screenshot 2023-10-06 at 12 16 33

## Link to Trello card

[](https://trello.com/c/hAs2umA8/653-review-terms-and-conditions-page-to-make-sure-the-content-lines-up-with-continuous-applications)

## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by MaeveRoseveare** [comment]
[Review Terms and conditions page to make sure the content lines up with continuous applications](https://trello.com/c/hAs2umA8/653-review-terms-and-conditions-page-to-make-sure-the-content-lines-up-with-continuous-applications)

File: app/views/content/terms_candidate.html.erb [hunk 1]
```diff
@@ -7,75 +7,90 @@
...
 
     <ul class="govuk-list govuk-list--bullet">
-      <li>submit an application to initial teacher training courses</li>
-      <li>make contact with other people using this service</li>
+      <li>apply to initial teacher training courses</li>
+      <li>contact other people using this service</li>
       <li>link to this service, or interact with it in any other way</li>
     </ul>
 
     <p class="govuk-body">Applying for teacher training using this service means you agree to these terms. If you do not agree, you must stop using the service.</p>
```
Comment (Kateharries) [inline app/views/content/terms_candidate.html.erb:15]: Should the first part of this sentence be a heading? 'Apply for teacher training (Apply)' then a new para: Using Apply means you agree to these terms...' Or should the sentence be 'By using Apply for teacher training (Apply) you are agreeing to these terms...'

**Review by Kateharries — COMMENTED** [review]
Just one comment from me :)

File: app/views/content/terms_candidate.html.erb [hunk 2]
```diff
@@ -7,75 +7,90 @@
...
 
     <p class="govuk-body">You must not:</p>
 
     <ul class="govuk-list govuk-list--bullet">
-      <li>use our service for illegal or fraudulent reasons</li>
-      <li>gather or use data from our service</li>
-      <li>use abusive language or act abusively towards anyone you come into contact with through using this service</li>
+      <li>use the service for illegal or fraudulent reasons</li>
+      <li>gather or use data from the service</li>
+      <li>use abusive language or act abusively towards anyone you contact using this service</li>
```
Comment (MylesJarvis) [inline app/views/content/terms_candidate.html.erb:32]: I worry this potentially weakens our standing if someone is abusive to someone in, say, an interview with a provider. If the provider fed this back, what would our recourse be to block this person? 

The former version gives us remit to take action, as they came into contact through the use of the service (applying and then attending the interview), whereas this new version risks not being able to, as the candidate doesn't use the service to directly contact the person.

What do you think?

**Comment by MylesJarvis** [comment]
Added one comment, but otherwise I think good to go ✅

File: app/views/content/terms_candidate.html.erb (hunk 2 above)
Comment (MaeveRoseveare) [reply app/views/content/terms_candidate.html.erb]: Sounds good, have updated

File: app/views/content/terms_candidate.html.erb (hunk 1 above)
Comment (MaeveRoseveare) [reply app/views/content/terms_candidate.html.erb]: I've tweaked this slightly.

**Review by MylesJarvis — APPROVED** [review]

**Review by JR-G — APPROVED** [review]

**Review by Kateharries — APPROVED** [review]
Looks good

**Review by inulty-dfe — APPROVED** [review]

**Pull request merged**